lib/src/tabs/tab-set.component.scss

```diff
@@ -1,17 +1,22 @@
 @import '../sass/cashmere';
 
+:host {
+    display: table;
+    width: 100%;
+}
+
 .tab-bar-vertical {
     padding-top: 15px;
     background-color: $white;
     width: 20%;
-    float: left;
-    height: calc(100% - 15px);
+    display: table-cell;
+    vertical-align: top;
```

Comment:
@corytak - the issue I was running into was that if the tab bar was taller than the content, the gray background did not extend to the entire height of the tab bar.  Basically it's the issue of trying to match two floating divs with variable heights.  I researched and tried a number of approaches, but using display: table seemed to be the best cross browser solution.  Works well on the demo site, but let me know if you think this is going to cause problems down the road.

Reply:
Works for me. 👍